.devcontainer/devcontainer.json

```diff
@@ -104,8 +97,8 @@
 // "overrideFeatureInstallOrder": [
 // 	"ghcr.io/devcontainers/features/common-utils:1"
 // ],
-"onCreateCommand": "dos2unix -n .devcontainer/createCommand.sh .devcontainer/createCommand_unix.sh && chmod +x .devcontainer/createCommand_unix.sh && bash .devcontainer/createCommand_unix.sh",
-"postStartCommand": "poetry install",
+"onCreateCommand": "sudo dos2unix -n .devcontainer/createCommand.sh .devcontainer/createCommand_unix.sh && sudo chmod +x .devcontainer/createCommand_unix.sh && bash .devcontainer/createCommand_unix.sh",
```

Comment:
why the last command it executed via bash if it's changed to be executed?
and why do you need the dos2unix? why not fix the file to begin with?
and why not make it executable in the git repository to begin with?

Reply:
1. checking if chmod is reduntant
2. we had issues when creating devcontainer from Windows machines where Windows automatically changed the `.devcontainer/createCommand.sh`, and if failed when running this script from the container.
3. I will try.

Reply:
1+3: dont need the chmod